### PR TITLE
doc: fix titles

### PR DESCRIPTION
**Pull request by lkppo**
Sic

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -1646,6 +1647,8 @@ println([1,2,3]) // "[1, 2, 3]"
 println(User{name:'Bob', age:20}) // "User{name:'Bob', age:20}"
 ```
 
+## Custom print of types
```
Comment (spytheman) [inline doc/docs.md:1650]: Would not `Custom printing of types` be better (I am not a native English speaker, so I may be wrong...) ?

Comment (Delta456) [reply doc/docs.md:1650]: I think `Custom Types Printing` sounds better here but lets check with @JalonSolov.

Comment (JalonSolov) [reply doc/docs.md:1650]: Hmm... I hadn't read this part of the docs closely before... The proposed change is to
```
## Custom print of types

If you want to define a custom print value for your type,
```
I would change that to
```
## Printing custom types

If you want your custom type to be printed in a different style than the default,
```

Comment (lkppo) [reply doc/docs.md:1650]: Both are custom, or user defined, and the first sentence is clear, so we can turn it on any side or be more precise like: "Customize the printing of user defined types". Choice the way you want and I will change my PR. My proposal was only the distinction of subjects.

Comment (spytheman) [reply doc/docs.md:1650]: @lkppo all right, please change it to the second variant that JalonSolov proposed.

Comment (lkppo) [reply doc/docs.md:1650]: Done

**Pull request merged**